types/matomo-tracker-react-native/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Type definitions for matomo-tracker-react-native 0.3
+// Project: https://github.com/donni106/matomo-tracker-react-native
+// Definitions by: Zeynep Ece Ergin <https://github.com/zecergin>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.4
```

Comment:
why this version? (which was just recently the 'top notch' version) what language feature depends on it here?